torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6022,14 +6025,6 @@ def method_tests():
         ('div', (), (uniform_scalar(0.1j),), 'complex_scalar_broadcast_lhs', (True,)),
         ('div', torch.rand(S, S, S, dtype=torch.cdouble) + 1e-1, (3.14j,), 'complex_constant', (True,)),
         ('div', uniform_scalar(1e-1j, requires_grad=True), (3.14j,), 'complex_scalar_constant', (True,)),
-        ('float_power', torch.rand(S, S, S) + 1e-3, (torch.rand(S, S, S) + 0.1,), ''),
-        ('float_power', torch.rand(S, S, S) + 1e-3, (torch.rand(1,) + 0.1,), 'broadcast_rhs'),
-        ('float_power', torch.rand(1,) + 1e-3, (torch.rand(S, S, S) + 0.1,), 'broadcast_lhs'),
-        ('float_power', torch.rand(S, 1, S) + 1e-3, (torch.rand(1, S, 1) + 0.1,), 'broadcast_all'),
-        ('float_power', uniform_scalar(1e-3, requires_grad=True), (uniform_scalar(0.1),), 'scalar'),
-        ('float_power', torch.rand(S, S, S) + 1e-3, (uniform_scalar(0.1),), 'scalar_broadcast_rhs'),
-        ('float_power', uniform_scalar(1e-3, requires_grad=True), (torch.rand(S, S, S) + 0.1,), 'scalar_broadcast_lhs'),
-        ('float_power', torch.rand(S, S, S) + 1e-3, (3.14,), 'constant'),
```

Comment:
Verified that all the cases are covered in (with different shapes but capture all the cases)
https://github.com/pytorch/pytorch/blob/126ea1ccad0d46d77b5225dcef16ca9b40d7964b/torch/testing/_internal/common_methods_invocations.py#L2373

@mruberry I am just curious to know, why the tensors in samples use explicitly use `(2, 2)` instead of `(S, S)` (is it because of the test times?)

Reply:
We were probably being extra sensitive to test timings for that PR, yes; there likely wasn't a lot of thought put into it

With @soulitzer's {grad, gradgrad}check time improvements, however, we could expand these sizes